alf/examples/sac_carla_conf.py

```diff
@@ -60,17 +63,50 @@
 
 alf.config(
     'CarlaEnvironment',
-    vehicle_filter='vehicle.audi.tt',
+    vehicle_filter='vehicle.*',
     num_other_vehicles=20,
     num_walkers=20,
     # 1000 second day length means 4.5 days in replay buffer of 90000 length
     day_length=1000,
     max_weather_length=500,
 )
 
-alf.define_config('taac', False)
+
+class NormalizedActionWrapper(AlfEnvironmentBaseWrapper):
+    """An AlfEnvironment wrapper that converts the last action dim ('reverse') from
+    [0,1] to [-1,1].
+    """
+
+    def __init__(self, env):
+        super().__init__(env)
+        self._action_spec = BoundedTensorSpec(
+            shape=env.action_spec().shape,
+            dtype=env.action_spec().dtype,
+            minimum=-1.,
+            maximum=1.)
+        self._time_step_spec = env.time_step_spec()._replace(
+            prev_action=self._action_spec)
+
+    def _step(self, action):
+        # in: [-1,1] -> [0,1]
+        action[..., -1] = (action[..., -1] + 1.) / 2.
+        return self._env.step(action)
```

Comment:
prev_action of the returned TimeStep is not correctly handled

Reply:
> prev_action of the returned TimeStep is not correctly handled

Correct. I think this will affect the training of TaacL and TaacQ. (Taac doesn't require action_spec to be [-1,1] and my paper experiments should not be affected). I'll rerun the current reproducing job.